taxcalc/tests/test_parameters.py

```diff
@@ -41,6 +32,226 @@ def test_create_parameters():
     assert p
 
 
+def test_multi_year_reform():
+    """
+    Test implementation of multi-year reform involving 1D and 2D parameters.
+    """
+    # specify dimensions of policy Parameters object
+    syr = 2013
+    nyrs = 10
+
+    # specify assumed inflation rates
+    irates = {2013: 0.02, 2014: 0.02, 2015: 0.02, 2016: 0.03, 2017: 0.03,
+              2018: 0.04, 2019: 0.04, 2020: 0.04, 2021: 0.04, 2022: 0.04}
+    ifactors = {}
+    for i in range(0, nyrs):
+        ifactors[syr + i] = 1.0 + irates[syr + i]
+    iratelist = [irates[syr + i] for i in range(0, nyrs)]
+
+    # instantiate policy Parameters object
+    ppo = Parameters(start_year=syr, budget_years=nyrs, inflation_rates=irates)
+
+    # confirm that parameters have current-law values
+    assert_array_equal(getattr(ppo, '_AMT_thd_MarriedS'),
+                       expand_array(np.array( #pylint: disable=no-member
+                           [40400, 41050]),
+                                    inflate=True, inflation_rates=iratelist,
+                                    num_years=nyrs))
+    assert_array_equal(getattr(ppo, '_EITC_c'),
+                       expand_array(np.array( #pylint: disable=no-member
+                           [[487, 3250, 5372, 6044],
+                            [496, 3305, 5460, 6143],
+                            [503, 3359, 5548, 6242]]),
+                                    inflate=True, inflation_rates=iratelist,
+                                    num_years=nyrs))
+    assert_array_equal(getattr(ppo, '_II_em'),
+                       expand_array(np.array( #pylint: disable=no-member
+                           [3900, 3950, 4000]),
+                                    inflate=True, inflation_rates=iratelist,
+                                    num_years=nyrs))
+    assert_array_equal(getattr(ppo, '_SS_Earnings_c'),
+                       expand_array(np.array( #pylint: disable=no-member
+                           [113700, 117000, 118500]),
+                                    inflate=True, inflation_rates=iratelist,
+                                    num_years=nyrs))
+
+    # specify multi-year reform using a dictionary of year_provisions dicts
+    reform = {
```

Comment:
@martinholmer, were you interested in organizing these dictionaries by parameter rather than by year. I.e, like the snippet below rather than what is shown in the test? 

```
reform = {
    '_AMT_thd_MarriedS': {
        _c_cpi': False
        2015: [60000],
        2016: [65000]
    },
    '_EITC_c': {
        2016: [[900, 5000, 8000, 9000]],
        2017: [[950, 6000, 8000, 9000]],
        2018: [[950, 6000, 8000, 1000]]
    }
}
```
This would parallel more closely, (and potentially be a better supplement, even alternative to) our current method of specifying multi-year reforms  {name: [1st_yr_val, 2nd_yr_val, 3rd_yr_val, ...]}